### ISSUE TITLE
An error 500 is shown if you do not select any file changing the profile image 

### ISSUE THREAD

**Issue by davitol** [owncloud/core#21533]
### Steps to reproduce

1. Go to personal settings page
2. In Profile picture section, click on 'Select from files button'
3. Click on choose button without choosing any file

### Expected behaviour

The 'choose an image window' should be closed without performing any action

### Actual behaviour

An error 500 is shown

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4.7

**Database:** mysql

**PHP version:** 5.5.9

**ownCloud version:  ownCloud Enterprise Edition 9.0 pre alpha (daily) Build:2016-01-07T04:44:34+00:00 d3922510d01fb5feed3ea01a3d390dfaa8b1312f

**Updated from an older ownCloud or fresh install:** Fresh install

**Are you using external storage, if yes which one:** no;
**Are you using encryption:** no
**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox

![screen shot 2016-01-08 at 08 54 35](https://cloud.githubusercontent.com/assets/10596560/12193546/a9c747f2-b5e9-11e5-8238-fb307db94ef1.png)

```
{"reqId":"sRz3Nl5twTSwAFfMqIaz","remoteAddr":"82.159.139.58","app":"PHP","message":"Call to undefined method OC\\Files\\Node\\Folder::getContent() at \/opt\/owncloud\/core\/avatar\/avatarcontroller.php#170","level":3,"time":"2016-01-08T07:54:25+00:00"}
```

**Comment by davitol** [comment]
@rullzer

**Comment by rullzer** [comment]
Ok got it I think :) We did not check if the supplied path was a valid file... PR incomming

**Comment by rullzer** [comment]
Btw, that red error bar looks way to big... @jancborchardt

**Comment by rullzer** [comment]
PR with fix in #21536

**Comment by jancborchardt** [comment]
@rullzer that error should show up as the yellow showNotification up top. :)
